staging/src/k8s.io/apimachinery/pkg/apis/meta/internalversion/types.go

```diff
@@ -66,6 +66,21 @@ type ListOptions struct {
 	// it does not recognize and will return a 410 error if the token can no longer be used because
 	// it has expired.
 	Continue string
+
+	// SendInitialEvents when set together with Watch option, begins the watch stream with synthetic "Added" events of all resources.
```

Comment:
nit: can you please try keeping the lines a bit shorter (not longer then the longest lines in this file)?